packages/expo/src/winter/fetch/fetch.ts

```diff
@@ -0,0 +1,47 @@
+import { requireNativeModule } from 'expo-modules-core';
+
+import { NetworkFetchError } from './FetchErrors';
+import { FetchResponse } from './FetchResponse';
+import { NativeRequest, NativeRequestInit, NativeResponse } from './NativeRequest';
+import { normalizeBodyInitAsync, normalizeHeadersInit } from './RequestUtils';
+import type { FetchRequestInit } from './fetch.types';
+
+const NetworkFetchModule = requireNativeModule('ExpoNetworkFetchModule');
+
+export async function fetch(url: string, init?: FetchRequestInit): Promise<FetchResponse> {
+  const response = new NetworkFetchModule.NativeResponse() as NativeResponse;
+  const request = new NetworkFetchModule.NativeRequest(response) as NativeRequest;
+
+  const headers = normalizeHeadersInit(init?.headers);
+
+  const { body: requestBody, overrideHeaders } = await normalizeBodyInitAsync(init?.body);
+  if (overrideHeaders) {
+    headers.push(...overrideHeaders);
+  }
```

Comment:
I'm uncertain about the behavior of `overrideHeaders`. Usually multiple HTTP headers with the same name are supported. iOS's documentation says that header values are concatenated using a comma separator. So `overrideHeaders` won't necessarily delete other headers with the same name.

IMO: "headers" should use `addValue()` and 
"overrideHeaders" should use `setValue()`.

Reply:
the `headers: string[][]` is an internal implementation that i passed from js to native. the overrideHeaders is for specific case while for FormData, we want to override the Content-Type header: https://github.com/expo/expo/blob/8fa3e8f9b892ab72aefa83fbac449425cbf8a507/packages/expo/src/winter/fetch/RequestUtils.ts#L121-L128

Reply:
If the developer specifies “content-type” in their request options, we want to completely override it and not append to it. That’s why I thought setValue() would be the correct method to call.

Reply:
good call. updated the overrideHeaders to replace existing header key